tools/EFToolsVsReferences.settings.targets

```diff
@@ -11,8 +11,10 @@
     <Reference Include="Microsoft.VisualStudio.Shell.Immutable.12.0" />
     <Reference Include="Microsoft.VisualStudio.TextTemplating.Interfaces.14.0" />
     <Reference Include="Microsoft.VisualStudio.TextTemplating.14.0" />
+    <Reference Include="Microsoft.VisualStudio.Telemetry" />
```

Comment:
We could also use the `Microsoft.VisualStudio.Telemetry` NuGet package, but referencing the private assembly directly seemed easier.